umap/static/umap/js/modules/schema.js

```diff
@@ -1,46 +1,61 @@
 import { translate } from './i18n.js'
 
+// Possible impacts
+// ['ui', 'data', 'limit-bounds', 'datalayer-index', 'remote-data', 'background']
+
 export const SCHEMA = {
-  zoom: {
-    type: Number,
-  },
-  scrollWheelZoom: {
+  browsable: {
+    impacts: ['ui'],
     type: Boolean,
-    label: translate('Allow scroll wheel zoom?'),
   },
-  scaleControl: {
+  captionBar: {
     type: Boolean,
-    label: translate('Do you want to display the scale control?'),
-    default: true,
+    impacts: ['ui'],
+    label: translate('Do you want to display a caption bar?'),
+    default: false,
+    impacts: [],
```

Comment:
Two different `impacts` keys for the captionBar?

Reply:
🤫